app/ui/src/app/integration/import-export/import/integration-import.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { PlatformStore } from '@syndesis/ui/platform';
+
+
+import {
+  IntegrationImportUpload
+} from './integration-import.actions';
+
+@Component({
+  selector: 'syndesis-import-integration-component',
+  templateUrl: './integration-import.component.html',
+  styleUrls: ['./integration-import.component.scss']
+})
+export class IntegrationImportComponent implements OnInit {
+  constructor(private store: Store<PlatformStore>) { }
+
+  ngOnInit() {
+    /**
+     * Payload for IntegrationImportUpload action
+     */
+    const payload = {
+      list: [],
+      file: null,
+      importResults: {
+        integrations: [],
+        connections: []
+      }
+    };
+
+
+    // We dispatch the desired action with the given payload
+    this.store.dispatch(new IntegrationImportUpload(payload));
```

Comment:
Do you believe you're gonna be needing to dispatch the `IntegrationImportUpload` with different payloads in the future. Otherwise there is no point to dispatch an action with an object payload that reflects and empty initial state. You already have `initialState` at the reducer for this, and you can even initialize empty instances of state within the reducer handlers if needed.

Here we're only duplicating code and action payloads should be as minimal as possible or even non-existing whereas possible.